src/sentry/static/sentry/app/components/modals/integrationDetailsModal.jsx

```diff
@@ -32,13 +32,7 @@ const defaultFeatureGateComponents = {
       ungatedFeatures: p.features,
       gatedFeatureGroups: [],
     }),
-  FeatureList: p => (
-    <ul>
-      {p.features.map((f, i) => (
-        <li key={i} dangerouslySetInnerHTML={{__html: p.formatter(f.description)}} />
-      ))}
-    </ul>
-  ),
+  FeatureList: p => <ul>{p.features.map((f, i) => <li key={i}>{f.description}</li>)}</ul>,
```

Comment:
Could probably destructure `description` here, but just kept with what it was doing